### ISSUE TITLE
How to change matrixdropdown column's header require template ? 

### ISSUE THREAD

**Issue by MugiwaraNoMehdi** [surveyjs/survey-library#1500]
### Are you requesting a feature, reporting a bug or asking a question?
asking a question

### What is the current behavior?
when I change survey.questionTitleTemplate like this "{title} {require}", if  I have a matrixdynamic with required columns, the header of the columns doesn't have the questionTitleTemplate. So how can I change the header's template of a matrixdynamic columns ? 

### What is the expected behavior?
the expected behavior is to have the survey.questionTitleTemplate working for matrixdynamic columns or a workaround... 

### How would you reproduce the current behavior (if this is a bug)?
https://plnkr.co/edit/Dcj7NFGOCLCBQ58kYUce?p=preview

**Comment by tsv2013** [comment]
I think you can use the [onAfterRenderQuestion](https://surveyjs.io/Documentation/Library/?id=surveymodel#onAfterRenderQuestion) event to patch column headers.

**Comment by stanislavefremov** [comment]
I am also interested in making this work. I set survey.questionTitleTemplate to "{no} {title} {require}" and it works for all questions except those in Matrix Dymamic. The same if for survey.questionErrorLocation which I've set to "bottom" and all questions in survey shows error under the question input except inputs in Matrix Dynamic. Is it something could be configured or is there any workarounds to apply mentioned properties to questions in Matrix?
